integration-tests/deployment/memory/job_client.go

```diff
@@ -121,19 +132,92 @@ func (j JobClient) ListNodeChainConfigs(ctx context.Context, in *nodev1.ListNode
 				P2PKeyBundle: &nodev1.OCR2Config_P2PKeyBundle{
 					PeerId: n.Keys.PeerID.String(),
 				},
-				OcrKeyBundle: &nodev1.OCR2Config_OCRKeyBundle{
-					BundleId:              n.Keys.OCRKeyBundle.ID(),
-					ConfigPublicKey:       common.Bytes2Hex(cpk[:]),
-					OffchainPublicKey:     common.Bytes2Hex(offpk[:]),
-					OnchainSigningAddress: n.Keys.OCRKeyBundle.OnChainPublicKey(),
-				},
+				OcrKeyBundle:     evmKeyBundle,
 				Multiaddr:        n.Addr.String(),
 				Plugins:          nil,
 				ForwarderAddress: ptr(""),
 			},
 		})
 	}
+	for _, selector := range n.Chains {
+		family, err := chainsel.GetSelectorFamily(selector)
+		if err != nil {
+			return nil, err
+		}
+		chainID, err := chainsel.ChainIdFromSelector(selector)
+		if err != nil {
+			return nil, err
+		}
+
+		if family == chainsel.FamilyEVM {
+			// already handled above
+			continue
+		}
 
+		var ocrtype chaintype.ChainType
+		switch family {
+		case chainsel.FamilyEVM:
+			ocrtype = chaintype.EVM
+		case chainsel.FamilySolana:
+			ocrtype = chaintype.Solana
+		case chainsel.FamilyStarknet:
+			ocrtype = chaintype.StarkNet
+		case chainsel.FamilyCosmos:
+			ocrtype = chaintype.Cosmos
+		case chainsel.FamilyAptos:
+			ocrtype = chaintype.Aptos
+		default:
+			panic(fmt.Sprintf("Unsupported chain family %v", family))
+		}
```

Comment:
there's a bunch of these type conversions but I'm not sure where to put the helper functions